Bear.java

```diff
@@ -0,0 +1,48 @@
+import java.awt.*;
+
+public class Bear extends Critter {
+    private int count;
+    private boolean polar;
+
+    public Bear(boolean polar) {
+        this.count = 0;
+        this.polar = polar;
```

Comment:
I assume this means polar bear? been a while since i used objects but pretty sure you could make a PollarBear inherit from Bear and just override color